src/test/java/com/endava/soa_5/cucumber/hook/Hooks.java

```diff
@@ -1,7 +1,7 @@
 package com.endava.soa_5.cucumber.hook;
 
 import io.cucumber.java.After;
-import org.junit.Before;
+import io.cucumber.java.Before;
 
 public class Hooks {
 
```

Comment:
there is the possibility to add scenario description when executing it, so you can try to add this to the code.

ex. https://stackoverflow.com/questions/23602953/how-to-get-name-of-scenario-in-cucumber-java